content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Moving cgroup v1 Support into Maintenance Mode"
+date: 2024-08-14
+slug: kubernetes-1-31-moving-cgroup-v1-support-maintenance-mode
```

Comment:
For SEO, how about:
```suggestion
slug: cgroup-v1-support-to-maintenance-mode
```

Reply:
Looking at other blogs I think we need to have the version mentioned somewhere.

Reply:
Nope; no need.